### PR TITLE
Improve db coverage

### PR DESCRIPTION
**Pull request by sidler-mozilla**

### CODE REVIEW

File: subhub/tests/test_db.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# 130     42    68%   79-81, 84-91, 107-109, 136, 139-144, 148-153, 156-164, 167-172
```
Comment (bowlofstew) [inline subhub/tests/test_db.py]: Do we want to keep the coverage comments?

Comment (sidler-mozilla) [reply subhub/tests/test_db.py]: i deleted them

**Comment by bowlofstew** [comment]
build failure.

**Comment by sidler-mozilla** [comment]
Closing until I get more dev help resolving issues with the db tests.

**Pull request closed without merging**